### ISSUE TITLE
Button: Color accessibility issue

### ISSUE THREAD

**Issue by gabriel-piltzer-trimble** [trimble-oss/modus-wc-2.0#634]
### Prerequisites

- [x] I have [searched](https://github.com/trimble-oss/modus-wc-2.0/issues?q=is%3Aissue) for duplicate or closed issues
- [x] I have read the [contributing guidelines](https://github.com/trimble-oss/modus-wc-2.0/blob/main/CONTRIBUTING.md)

### Describe the issue

The Button component with these properties (`color: secondary` and `variant: outlined`) does not meet minimum color contrast ration thresholds.

<img width="2544" height="1176" alt="Image" src="https://github.com/user-attachments/assets/bcefdd3e-e383-4068-ba0d-61d3b14e78bd" />

### Reduced test cases

1. Install axe DevTools Chrome browser extension
2. Open [Button](https://trimble-oss.github.io/modus-wc-2.0/prod/?path=/docs/components-button--docs&globals=theme:modus-modern-light) page with theme `modus-modern-light`
3. Select the secondary color and outlined variant
4. Open axe DevTools and run a full page scan

### What is the issue regarding?

@trimble-oss/modus-wc

### Priority

Medium

### What product/project are you using Modus Web Components for?

Prism, AI Finance Assistant

### What is your team/division name?

CMS

### Are you willing to contribute?

Yes
